SQL/CreateDB.sql

```diff
@@ -1,44 +1,54 @@
-CREATE TABLE tab_valuta(
-    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
-    name TEXT,
-    code TEXT
+CREATE TABLE tab_currency(
+    currency_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    name TEXT NOT NULL,
+    code TEXT NOT NULL
 );
 
-CREATE TABLE tab_category_refill(
-    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
-    name TEXT
+CREATE TABLE tab_category_income(
+    category_income_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    name TEXT NOT NULL
 );
 
 CREATE TABLE tab_category_expense(
-    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
-    name TEXT
+    category_expense_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    name TEXT NOT NULL
 );
 
 CREATE TABLE tab_wallets(
-    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
-    name TEXT,
-    amount REAL,
-    id_valuta INTEGER,
-    FOREIGN KEY (id_valuta)  REFERENCES tab_valuta (id)
+    wallet_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    name TEXT NOT NULL,
+    balance REAL NULL,
+    currency_id INTEGER NOT NULL,
+    FOREIGN KEY (currency_id) REFERENCES tab_currency (currency_id)
+        ON DELETE NO ACTION
+        ON UPDATE NO ACTION
 );
 
-CREATE TABLE tab_reffiling(
-    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
-    date_Time TEXT,
-    id_wallet INTEGER,
-    id_category_refill INTEGER,
-    sum INTEGER,
-    FOREIGN KEY (id_category_refill)  REFERENCES tab_category_refill (id),
-    FOREIGN KEY (id_wallet)  REFERENCES tab_wallets (id)
+CREATE TABLE tab_incoming(
+    income_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    date_Time TEXT NOT NULL,
```

Comment:
```suggestion
    date_time TEXT NOT NULL,
```

В SQL принято именовать в стиле **_snake_case_**